src/jit/jit.h

```diff
@@ -160,9 +160,22 @@
 #error Unsupported or unset target architecture
 #endif
 
-#if defined(_TARGET_AMD64_) || defined(_TARGET_ARM64_)
+#if (defined(_TARGET_AMD64_) || defined(_TARGET_ARM64_)) && !defined(_TARGET_64BIT_)
+#ifndef FEATURE_CORECLR
 #define _TARGET_64BIT_
-#endif
+#else // FEATURE_CORECLR
+#error When _TARGET_AMD64_ or _TARGET_ARM64_ is defined you must define _TARGET_64BIT_ as well.
+#endif // FEATURE_CORECLR
```

Comment:
I don't remember if we still do `FEATURE_CORECLR` builds on the TFS side; if so, this will cause a build break. Seems like you could just unconditionally define `_TARGET_64BIT_` here. Not sure the error adds much.

Reply:
@BruceForstall I do as you suggested earlier today and keep consistency checks for 32-bit targets. Thanks